brokers/che-plugin-broker/broker/broker.go

```diff
@@ -133,7 +169,7 @@ func (cheBroker *ChePluginBroker) processPlugin(meta model.PluginMeta) error {
 	}
 
 	cheBroker.PrintDebug("Resolving '%s:%s'", meta.ID, meta.Version)
-	err = cheBroker.resolveToolingConfig(&meta, pluginPath)
+	err = cheBroker.resolveToolingConfig(meta, pluginPath)
```

Comment:
Can you elaborate on why this change is needed?

Reply:
In the current branch, ```meta``` parameter declared in function as  ```processArchive(meta *model.PluginMeta, url string)``` in master it's declared as ```processPlugin(meta model.PluginMeta)```.

Reply:
OK, thanks for the clarification